jwst/assign_wcs/nirspec.py

```diff
@@ -211,7 +219,19 @@ def ifu(input_model, reference_files, slit_y_range=[-.55, .55]):
 
     det, sca, gwa, slit_frame, msa_frame, oteip, v2v3, world = create_frames()
 
-    if input_model.meta.instrument.filter != 'OPAQUE':
+    exp_type = input_model.meta.exposure.type.upper()
+
+    is_lamp_exposure = exp_type in ['NRS_LAMP', 'NRS_AUTOWAVE', 'NRS_AUTOFLAT']
```

Comment:
There is a number of copies of this statement. Would this be appropriate to place in the `exptypes` library as function?

Reply:
Yes, I did make a bunch of boolean functions in exptypes, but this code was written before I did that and I didn't update it